src/web/components/ClientSideTokenGeneration/CstgAddDialog.spec.tsx

```diff
@@ -163,7 +163,7 @@ describe('CstgAddDialog', () => {
         onOpenChange={() => {}}
         existingCstgValues={[]}
         cstgValueType={CstgValueType.MobileAppId}
-        addInstructions='Please register the Android App ID, iOS/tvOS Bundle ID and iOS App Store ID.'
```

Comment:
This string is no longer accurate and isn't even needed. Figured I would remove all of them in this file